tests/integration/test_bayesian_optimization.py

```diff
@@ -336,12 +335,11 @@ def _test_optimizer_finds_minimum(
         State[TensorType, AsynchronousRuleState | TrustRegion.State], Box, GPflowPredictor
     ],
     optimize_branin: bool = False,
-    model_type: str = "GPR",  # in Python 3.8+ this could be Literal["GPR", "VGP", "SVGP"]
-    model_args: Optional[Mapping[str, Any]] = None,
+    model_type: str = "GPR",  # in Python 3.8+ a Literal["GPR", "VGP", "SVGP", "DGP", "DE"]?
+    model_args: Mapping[str, Any] = {},
```

Comment:
You might get a warning here for having a mutable default argument. And because Python doesn't have a built in frozendict class it's probably best to leave it as an optional defaulting to None.

Reply:
I think it would still be better to keep the immutable default as it's both more Pythonic and less error prone.

Reply:
ok, done!